SPECS/pciutils/pciutils.spec

```diff
@@ -1,26 +1,28 @@
-Summary:	System utilities to list pci devices
-Name:		pciutils
-Version:	3.6.2
-Release:        4%{?dist}
-License:	GPLv2
-URL:		https://www.kernel.org/pub/software/utils/pciutils/
-Group:		System Environment/System Utilities
+Summary:        System utilities to list pci devices
+Name:           pciutils
+Version:        3.7.0
+Release:        1%{?dist}
+License:        GPLv2
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
-Source0:	https://www.kernel.org/pub/software/utils/pciutils/%{name}-%{version}.tar.gz
+Group:          System Environment/System Utilities
+URL:            https://www.kernel.org/pub/software/utils/pciutils/
+Source0:        https://www.kernel.org/pub/software/utils/%{name}/%{name}-%{version}.tar.gz
 
 %description
 The pciutils package contains a set of programs for listing PCI devices, inspecting their status and setting their configuration registers.
 
 %package devel
-Summary: Development Libraries for openssl
-Group: Development/Libraries
-Requires: pciutils = %{version}-%{release}
+Summary:        Development Libraries for openssl
```

Comment:
Ha, looks like this summary got copied from the openssl spec at some point in the past. Mind changing this to something more accurate to this package?